packages/shared-ux/page/solution_nav/src/solution_nav.tsx

```diff
@@ -77,14 +74,12 @@ const setTabIndex = (items: Array<EuiSideNavItemType<{}>>, isHidden: boolean) =>
   });
```

Comment:
@cchaos Can I remove this shim?  https://github.com/elastic/eui/issues/4925

Reply:
I'd have to do some local testing to make sure and I'm not sure I have time today

Reply:
No rush, I'll keep my eye on it.  I tested it and didn't see an issue.